pkg/processor/trigger/rabbitmq/factory.go

```diff
@@ -50,7 +50,7 @@ func (f *factory) Create(parentLogger logger.Logger,
 	workerAllocator, err := f.GetWorkerAllocator(triggerConfiguration.WorkerAllocatorName,
 		namedWorkerAllocators,
 		func() (eventprocessor.Allocator, error) {
-			return worker.WorkerFactorySingleton.CreateNonBlockingWorkerAllocator(triggerLogger, configuration.NumWorkers,
+			return worker.WorkerFactorySingleton.CreateFixedPoolWorkerAllocator(triggerLogger, configuration.NumWorkers,
```

Comment:
What does that mean regarding asynchronous work?

Reply:
@TomerShor it means no async work, because if we want to be able to get response from the wrapper with an error code (info about exception was raised), we have to work in a sync mode. However, for rabbitmq trigger it will be possible to enable async mode because of ack logic based on messages, not on partitions.